breakpad-symbols/src/sym_file/types.rs

```diff
@@ -96,10 +119,7 @@ impl StackInfoCfi {
         if self.size == 0 {
             return None;
         }
-        Some(Range::new(
-            self.init.address,
-            self.init.address.checked_add(self.size as u64)? - 1,
-        ))
+        Some(self.init.address..self.init.address.checked_add(self.size as u64)?)
```

Comment:
this one stays checked while all the others saturate?

Reply:
Hmm, true, it's inconsistent that I kept all the methods which return ranges to use checked, and then used saturate in `end_address()`. The methods which return a range want to return None if the range is empty or inverted, so using checked is easier there.